### ISSUE TITLE
Destiny build / Bioconductor Devel?

### ISSUE THREAD

**Issue by bbimber** [theislab/destiny#43]
hello - my lab has an R package that depends on destiny, pulling form bioconductor.  our build off bioconductor devel is failing b/c it cannot download destiny. I'm not that familiar with how packages are pushed to bioconductor; however, i see your bioconductor builds are failing due to a latex/inputenc issue. I assume this is why destiny is not available to pull from bioconductor/devel? is this an issue you plan to investigate? Thanks.

**Comment by rlarrosa1** [comment]
True, it is still broken, and it is important that destiny can be easily installed from bioconductor.

Thanks.
